src/Microsoft.Diagnostics.Tracing.Eventing/EventPipeClient.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>Microsoft.Diagnostics.Tracing.Eventing</RootNamespace>
```

Comment:
These initial commands are for EventPipe, but I think of this library as a general purpose remoting client for everything in the DiagnosticsService. For example it might soon have profiler attach APIs or cooperative dump writing APIs.

Perhaps name it Microsoft.Diagnostics.Tools.RuntimeClient?

Reply:
I like it. Will do! :)

Reply:
This is a nit, but the other assemblies are not plural; they start with "Microsoft.Diagnostic.". 

I think I messed up with I named the first one Microsoft.Diagnostic.TestHelpers and to be consist I named the newer one Microsoft.Diagnostic.Repl.  We should be consistent one way or another.

Reply:
I tried using the `Microsoft.Diagnostics` namespace we use in tools like [Perfview](https://github.com/Microsoft/perfview/blob/master/src/TraceEvent/TraceEvent.cs#L7), but I'm open to change it. I'm not fixed on any name yet and I'm open to suggestions :)